app/components/file/browse/display/file-list-display-filter.pipe.ts

```diff
@@ -6,6 +6,8 @@ import { File } from "app/models";
 @Pipe({name: "fileDirectoryFilter"})
 export class FileDirectoryFilter implements PipeTransform {
     public transform(files: List<File>, args: any[]): any {
-        return files.filter(file => file.isDirectory === false);
+        // TODO: Tim is this ok? files is null when not using an RxProxy (it would seem ...)
```

Comment:
you should not have to do this here. THis is problably there is no inital value for the list `public myList: List<File>;` instead of `public myList: List<File> = List([])`